### ISSUE TITLE
ability to crete mask from points file

### ISSUE THREAD

**Issue by SigireddyBalasai** [tensorflow/tensorflow#62719]
### Issue type

Documentation Feature Request

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

source

### TensorFlow version

latest

### Custom code

Yes

### Current behavior?

i want to know if there is a way to get a label from bbox coordinates

### Standalone code to reproduce the issue

```shell
i dont have one
```

**Comment by SuryanarayanaY** [comment]
Hi @SigireddyBalasai ,

I am unable to get your query. Could you please elaborate ?

**Comment by SigireddyBalasai** [comment]
i know it is present in other libraries but i am asking in tf

**Comment by SigireddyBalasai** [comment]
Basically I am doing a image segmentation project and I need to create a
polygon mask from points file and there are quite a methods in other
modules I want to see if tensor flow has such a method if not I am
suggesting to implement is hope this is clear

**Comment by SigireddyBalasai** [comment]
@SuryanarayanaY

**Comment by SuryanarayanaY** [comment]
Hi @SigireddyBalasai ,

Let me put my understandings here.

> i want to know if there is a way to get a label from bbox coordinates
 
For generating labels from bounding boxes you might need a backbone model that can able to detect the objects in the bounding box.You may refer this [example](https://keras.io/examples/vision/sam/) for this.




> Basically I am doing a image segmentation project and I need to create a polygon mask from points file and there are quite a methods in other modules I want to see if tensor flow has such a method if not I am suggesting to implement is hope this is clear

If you want to generate a bbox you may use the [draw_bounding_boxes](https://www.tensorflow.org/api_docs/python/tf/image/draw_bounding_boxes) API.

If you want to generate a mask for the bbox,as per my understanding its not pretty straight forward with a single method/API.You can refer this [tutorial](https://www.tensorflow.org/tfmodels/vision/instance_segmentation) for an implementation.

As you are referring there are methods from other libraries, could you please refer some examples so that it can be better understood and can provide a appropriate response.

Thanks!

**Comment by SuryanarayanaY** [comment]
Hi @SigireddyBalasai ,

Can you add some references of other libraries for better understanding. Would you like to contribute to this feature?

**Comment by SuryanarayanaY** [comment]
Hi @SigireddyBalasai ,

I am not 100% getting the requirement. Add a sample demo or equivalent API from other libraries(if any) so that we can take a call on that. Thanks!
